src/main/java/com/cactus/adapters/ClassAuthAdaptor.java

```diff
@@ -0,0 +1,45 @@
+package com.cactus.adapters;
+
+import com.cactus.data.EntityRepository;
+import com.cactus.entities.User;
+import java.util.Collection;
+import java.util.HashMap;
+
+/***
+ * Represents ClassAuthAdapter which implements AuthAdapter interface.
+ */
+
+public class ClassAuthAdaptor implements AuthAdapter{
+
+    public EntityRepository repository;
```

Comment:
Perhaps this attribute should be private? The point is that no other class other than the adapters should be able to access the EntityRepository.